packages/client/src/app/components/site-sidenav/notifications/notif-item/notif-item.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="notif-item">
+    <a class="notification-link" (click)="sendTo()">
```

Comment:
Can this be a real link? If it's a "let's do it later", make an issue for it

Reply:
that would introduce more `ng-container`s and `ngSwitch`es and the reason notifications got moved to their own subcomponent was to combat an issue where too many was preventing certain elements from even *appearing* on the page

Reply:
fair, I guess. but I _guarantee_ this is gonna piss users off